xbmc/settings/dialogs/GUIDialogAudioDSPManager.cpp

```diff
@@ -1026,14 +1026,17 @@ CFileItem *CGUIDialogAudioDSPManager::helper_CreateModeListItem(CActiveAEDSPMode
   // set list item properties
   pItem->SetProperty("ActiveMode", isActive);
   pItem->SetProperty("Number", number);
-  pItem->SetProperty("Name", modeName);
+  pItem->SetLabel2(modeName);
```

Comment:
Looks good, but maybe swap Label2 with Label, the Mode name is more the primary in DSP Manager.